src/tests/tribol_mfem_mortar_lm.cpp

```diff
@@ -223,6 +247,10 @@ int main( int argc, char* argv[] )
   axom::slic::SimpleLogger logger;  // create & initialize test logger, finalized when
                                     // exiting main scope
 
+#ifdef TRIBOL_ENABLE_CUDA
+  mfem::Device device( "cuda" );
```

Comment:
Confirm that we'd only run on cpu or cuda.

Reply:
Actually fixed now.  Forgot for a second there we don't support GPU for mortar.